config/locales/en.yml

```diff
@@ -10,7 +10,12 @@ en:
       hint_stub_creation_html: <span>Hint:</span>To see your component make sure you've created stubs
       tip_stub_creation_html: 'You should write stub examples here:'
     index:
-      description: Select one of the components from the side to view its examples and documentation.
+      description_html:
+        <p>Select one of the components from the side to view its examples and documentation.</p>
+        <br />
+        <br />
+        <p>To replace the content on this page, add the directory `app/views/mountain_view` in your application,
+        and create a partial named <code class="language-ruby">_main_page_content.html.erb</code> (or haml, slim, etc).</p>
```

Comment:
No need to add `class="language-ruby"` for this, I think. It's the name of the file, nor a ruby code block.

Reply:
Removed it :) replaced with ` to match the directory on the previous line.